src/app/course/event/event-row/event-row.component.html

```diff
@@ -118,6 +118,21 @@
                 </button>
             </div>
         </ng-template>
+        <button *ngIf="event.has_edit_permission"
+                (click)="openDeleteEventDialog(deleteDialog, event.id)"
+                [tuiHint]="deleteTooltip"
+                class="tui-space_right-2 tui-space_vertical-2"
```

Comment:
use tailwind classes

Reply:
All the other buttons use the tui spacing, I fixed it on this one but do we want to add a task to trello to change them over to tailwind?